philutku2.c

```diff
@@ -0,0 +1,239 @@
+/*
+
+  _____  _       _                                           
+ |  __ \(_)     (_)                                          
+ | |  | |_ _ __  _ _ __   __ _                               
+ | |  | | | '_ \| | '_ \ / _` |                              
+ | |__| | | | | | | | | | (_| |                              
+ |_____/|_|_| |_|_|_| |_|\__, |          _                   
+ |  __ \| |   (_) |       __/ |         | |                  
+ | |__) | |__  _| | ___  |___/___  _ __ | |__   ___ _ __ ___ 
+ |  ___/| '_ \| | |/ _ \/ __|/ _ \| '_ \| '_ \ / _ \ '__/ __|
+ | |    | | | | | | (_) \__ \ (_) | |_) | | | |  __/ |  \__ \
+ |_|    |_| |_|_|_|\___/|___/\___/| .__/|_| |_|\___|_|  |___/
+                                  | |                        
+                                  |_| 
+
+Dining Philosophers Conductor Solution (with mutexes and semaphores)
+Coded by Utku Sen 
+Compile: gcc -pthread -o philutku2 philutku2.c
+
+*/
+
+

```

Comment:
Add #include <stdio.h>
```suggestion

#include <stdio.h>
```